main.css

```diff
@@ -0,0 +1,142 @@
+/* General reset for margin and padding */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+.hero_container {
+    display: flex;
+    flex-direction: row;  
+    padding: 6vw;  
+    gap: 1vw;
+}
+
+.left-col {
+    flex: 2;  
+    line-height: 1.2;
+    padding: 10px;
+}
+
+.right-col {
+    flex: 1; 
+    padding: 10px;
+}
+
+.pic{
```

Comment:
Nitpick: be consistent with the spacing ".pic{" change to ".pic {"

Do this for the rest. There's an autoformatting function we can show you in VSCode